### DISCUSSION TITLE
Website update / refresh is needed

### DISCUSSION THREAD

**Discussion by xemjeff** [wren-lang/wren#1136] [General]
The complete example in embedded/complete example on the wren.io website is not up to date with the github repo, and this causes a build error (missing <stdio.h>.  I did not open an issue for this since the repo is correct.

Website url: `https://github.com/wren-lang/wren/blob/main/doc/site/embedding/index.markdown`

```
//For more details, visit https://wren.io/embedding/

#include 
#include "wren.h"

```
Github (correct)

```
//For more details, visit https://wren.io/embedding/

#include <stdio.h>
#include "wren.h"

```

**Comment by ruby0x1** [comment]
Yea it's coming soon, thanks for the report. We're gonna run the doc updates automatically on github actions (it was on travis before).

Sorry for the inconvenience!
